lang/cs/Org.Apache.REEF.Client/YARN/YarnCommandLineEnvironment.cs

```diff
@@ -87,7 +96,7 @@ internal string GetYarnCommandPath()
         /// </summary>
         /// <param name="arguments"></param>
         /// <returns>Whatever was printed to stdout by YARN.</returns>
-        internal string Yarn(params string[] arguments)
+        private string Yarn(params string[] arguments)
```

Comment:
@anupam128 @markusweimer Actually, I'm not so sure why this needs to be used for this PR. I thought this PR was specifically for REST submission? Maybe I'm missing something, but this doesn't seem like it would be able to work if you weren't already submitting from the cluster.